### ISSUE TITLE
Duplicate upload after deleting a photo from the Immich web app

### ISSUE THREAD

**Issue by rafolinux** [immich-app/immich#14165]
### The bug

When the application is configured to upload files from a specific folder, it works as expected for the initial upload. However, if I delete a photo from the Immich web interface, during the next scan, the application identifies the file as missing on the server and re-uploads it, even though it is already present in the source folder.

This behavior causes duplicate uploads of the same file, which is not desirable.

### The OS that Immich Server is running on

debain12

### Version of Immich Server

v1.120.2

### Version of Immich Mobile App

v1.120.2

### Platform with the issue

- [ ] Server
- [X] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
name: immich

services:
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:${IMMICH_VERSION:-release}
    # extends:
    #   file: hwaccel.transcoding.yml
    #   service: cpu # set to one of [nvenc, quicksync, rkmpp, vaapi, vaapi-wsl] for accelerated transcoding
    volumes:
      # Do not edit the next line. If you want to change the media storage location on your system, edit the value of UPLOAD_LOCATION in the .env file
      - ${UPLOAD_LOCATION}:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
    env_file:
      - stack.env
    ports:
      - '2283:2283'
    depends_on:
      - redis
      - database
    restart: always
    healthcheck:
      disable: false

  immich-machine-learning:
    container_name: immich_machine_learning
    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
    # Example tag: ${IMMICH_VERSION:-release}-cuda
    image: ghcr.io/immich-app/immich-machine-learning:${IMMICH_VERSION:-release}
    # extends: # uncomment this section for hardware acceleration - see https://immich.app/docs/features/ml-hardware-acceleration
    #   file: hwaccel.ml.yml
    #   service: cpu # set to one of [armnn, cuda, openvino, openvino-wsl] for accelerated inference - use the `-wsl` version for WSL2 where applicable
    volumes:
      - model-cache:/cache
    env_file:
      - stack.env
    restart: always
    healthcheck:
      disable: false

  redis:
    container_name: immich_redis
    image: docker.io/redis:6.2-alpine@sha256:2ba50e1ac3a0ea17b736ce9db2b0a9f6f8b85d4c27d5f5accc6a416d8f42c6d5
    healthcheck:
      test: redis-cli ping || exit 1
    restart: always

  database:
    container_name: immich_postgres
    image: docker.io/tensorchord/pgvecto-rs:pg14-v0.2.0@sha256:90724186f0a3517cf6914295b5ab410db9ce23190a2d9d0b9dd6463e3fa298f0
    environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
      POSTGRES_INITDB_ARGS: '--data-checksums'
    volumes:
      # Do not edit the next line. If you want to change the database storage location on your system, edit the value of DB_DATA_LOCATION in the .env file
      - ${DB_DATA_LOCATION}:/var/lib/postgresql/data
    healthcheck:
      test: pg_isready --dbname='${DB_DATABASE_NAME}' --username='${DB_USERNAME}' || exit 1; Chksum="$$(psql --dbname='${DB_DATABASE_NAME}' --username='${DB_USERNAME}' --tuples-only --no-align --command='SELECT COALESCE(SUM(checksum_failures), 0) FROM pg_stat_database')"; echo "checksum failure count is $$Chksum"; [ "$$Chksum" = '0' ] || exit 1
      interval: 5m
      start_interval: 30s
      start_period: 5m
    command:
      [
        'postgres',
        '-c',
        'shared_preload_libraries=vectors.so',
        '-c',
        'search_path="$$user", public, vectors',
        '-c',
        'logging_collector=on',
        '-c',
        'max_wal_size=2GB',
        '-c',
        'shared_buffers=512MB',
        '-c',
        'wal_compression=on',
      ]
    restart: always

volumes:
  model-cache:
```

### Your .env content

```Shell
environment:
      POSTGRES_PASSWORD: ${DB_PASSWORD}
      POSTGRES_USER: ${DB_USERNAME}
      POSTGRES_DB: ${DB_DATABASE_NAME}
      POSTGRES_INITDB_ARGS: '--data-checksums'
```

### Reproduction steps

1. Open the Immich mobile application and configure it to upload photos from a specific folder on your device.
2. Let the application perform the initial upload successfully.
3. Go to the Immich web interface and manually delete one of the uploaded photos.
4. Wait for the next scheduled scan by the mobile application or trigger a manual sync.
5. Observe that the photo deleted from the web interface is re-uploaded by the app, even though it has not been modified or re-added to the source folder.

**Comment by bo0tzz** [comment]
This is currently working as expected. You can prevent it by doing the delete from the app and choosing the option to also remove it from the phone. The behaviour you're looking for has been requested in #3594.

**Comment by zarembo** [comment]
I think there is still something wrong. What I did:

1. delete the file from my mobile phone and empty trash
2. delete the file in the web front end and also empty trash
3. reupload the image from my Notebook using the Web UI

Result: The image is still shown as a duplicate! I can even navigate to and view it, but why?! It must not be available anywhere.
I also tried to delete it after navigating to it (Information is shown that it was successful), but if I reupload it again (trash is still empty) it is still shown as a duplicate.
